### DISCUSSION TITLE
MSPT has risen. help me🙇‍♂️

### DISCUSSION THREAD

**Discussion by RangFarm** [BG-Software-LLC/SuperiorSkyblock2#1430] [Pending Feature Requests]
Recently, MSPT has risen due to a large number of users. I checked with Spark and I want to know what the problem is
Thank you for always helping the server. 🙇‍♂️

https://spark.lucko.me/qydhelLx7J

**Comment by OmerBenGera** [comment]
Open an issue under the issues tab please.

**Comment by OmerBenGera** [comment]
Open an issue under the issues tab please.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
